### PR TITLE
hotfix for the failing test, fixes #136

### LINKED ISSUE #136: fix failing test

### PR DESCRIPTION
**Pull request by TMRert**
Fixes the failing test in Application, which failed because of the modification of the path where levels can be found

### CODE REVIEW

File: src/test/java/net/liquidpineapple/pang/ApplicationTest.java [hunk 1]
```diff
@@ -45,6 +48,7 @@ public void testNotInCheatMode() throws Throwable {
 
   @After
   public void tearDown() throws Exception {
+    ScoreSystem.setScore(0);
```
Comment (grappigegovert) [inline src/test/java/net/liquidpineapple/pang/ApplicationTest.java]: Is this needed?

Comment (TMRert) [inline src/test/java/net/liquidpineapple/pang/ApplicationTest.java]: jup, because otherwise the score of the scoreSystem is not reset, leading to errors

Comment (grappigegovert) [inline src/test/java/net/liquidpineapple/pang/ApplicationTest.java]: It seems more logical to me to reset it at the start instead of at the end

**Comment by Zerochaosz** [comment]
LGTM

File: src/test/java/net/liquidpineapple/pang/ApplicationTest.java (hunk 1 above)
Comment (TMRert) [inline src/test/java/net/liquidpineapple/pang/ApplicationTest.java]: Sure, i'll change it to the @before then

**Pull request merged**